### PR TITLE
Add extra test case for bonfire Drop It

### PR DESCRIPTION
**Pull request by nscott**
Currently I can't get the build to work on my machine with the latest versions of mongo and node. 

`node seed/` fails with an OAuth problem.

That said, this test should fix https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2533

### CODE REVIEW

**Comment by BerkeleyTrue** [comment]
Please remove issue numbers from title

see [Pull-Request-Contribute](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute)

**Comment by nscott** [comment]
@BerkeleyTrue removed, sorry about that.

File: seed/challenges/intermediate-bonfires.json [hunk 1]
```diff
@@ -639,7 +639,8 @@
       "tests": [
         "expect(drop([1, 2, 3, 4], function(n) {return n >= 3; })).to.eqls([3, 4]);",
         "expect(drop([1, 2, 3], function(n) {return n > 0; })).to.eqls([1, 2, 3]);",
-        "expect(drop([1, 2, 3, 4], function(n) {return n > 5; })).to.eqls([]);"
+        "expect(drop([1, 2, 3, 4], function(n) {return n > 5; })).to.eqls([]);",
+        "expect(drop([1, 2, 3, 7, 4], function(n) { return > 5; })).to.eqls([7, 4]);"
```
Comment (ahstro) [inline seed/challenges/intermediate-bonfires.json]: Would you mind amending away the space before `return` here?

**Comment by ahstro** [comment]
This looks good, but I'm a bit hesitant to merge because of the errors I'm getting, `expected expression, got '>'` in Firefox and `Unexpected token >` in Chromium. Does anyone know the hows and whys of these error messages?

**Comment by BerkeleyTrue** [comment]
@ahstro Those may be unrelated. Sounds like something weird is going on when you are building.

**Comment by ahstro** [comment]
@BerkeleyTrue I tried re-pulling and rebuilding, but still there. The test seem to work fine, so I can merge if it's not something you're worried about, but it'd feel safer if someone else tested it as well.

File: seed/challenges/intermediate-bonfires.json (hunk 1 above)
Comment (BerkeleyTrue) [inline seed/challenges/intermediate-bonfires.json]: I'd prefer the space as it promotes readability and fits inline with our code style. But, it does break the pattern that is already there. I think this should not hold off merging this PR.

Comment (ahstro) [inline seed/challenges/intermediate-bonfires.json]: I prefer i too, but like you say, it was breaking pattern. I'll go ahead and merge this if it's not a problem though.

Comment (BerkeleyTrue) [inline seed/challenges/intermediate-bonfires.json]: just noticed the issue. `return > 5` is missing `n`.

**Comment by BerkeleyTrue** [comment]
merged! 20c45e7

**Pull request closed without merging**